### PR TITLE
Adding alias and some resolution fixes

### LINKED ISSUE #9: Add alias
Create alias to improve imports

### PR DESCRIPTION
**Pull request by luanraithz**
#9

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -2,5 +2,13 @@
   "presets": [
     "env",
     "react"
+  ],
+  "plugins": [
+    ["module-resolver", {
+      "alias": {
+        "colors": "./src/styleguide/colors.js",
+        "breakpoints": "./src/styleguide/breakpoints.js"
```
Comment (vmarcosp) [inline .babelrc]: what you think about create some groups like `styleguide/` and import something like colors using 
```js
import { blue } from 'styleguide/colors'
```
or
```js
import { mobileScreenSize } from 'styleguide/breakpoints'
```

Comment (luanraithz) [reply .babelrc]: That was my first thought, and the first thing that I've made, but I preferred to take them as separated modules. But you prefer, I may change it.

Comment (vmarcosp) [reply .babelrc]: I prefer separated modules, its more clean and organized :smile:

Comment (luanraithz) [reply .babelrc]: Okay

**Comment by vmarcosp** [comment]
Great pr @luanraithz <3

**Pull request merged**